src/ColumnsPaginatedBookView.ts

```diff
@@ -143,7 +166,7 @@ export default class ColumnsPaginatedBookView implements PaginatedBookView {
         this.setLeftColumnsWidth(0);
 
         const width = this.getColumnWidth();
-        const rightWidth = this.getRightColumnsWidth();
+        let rightWidth = this.getRightColumnsWidth();
```

Comment:
I think maybe this was an inadvertant change. It can be left as `const`, it seems.